bundles/org.openhab.binding.gardena/src/main/java/org/openhab/binding/gardena/internal/GardenaSmartWebSocket.java

```diff
@@ -70,7 +70,8 @@ public GardenaSmartWebSocket(GardenaSmartWebSocketListener socketEventListener,
         this.token = token;
         this.socketId = socketId;
 
-        webSocketClient = webSocketFactory.createWebSocketClient(socketId);
+        String webSocketId = String.valueOf(this.getClass().hashCode());
```

Comment:
Doesn't this give the same hashcode if you open 2 websockets because it takes the hashcode of the class itself? Maybe use hashcode of socketId?

Reply:
It's the hashCode of the instance and not of the class

Reply:
If you wanted the hashcode of the instance then you should doing this instead.
```suggestion
        String webSocketId = String.valueOf(hashCode());
```

Reply:
Good catch, @Hilbrand!

Reply:
Ups ... you are right, sorry!